src/routes/github/create-branch/github-create-branch-get.test.ts

```diff
@@ -0,0 +1,52 @@
+import express, { Application } from "express";
+import supertest from "supertest";
+import { getLogger } from "config/logger";
+import { getFrontendApp } from "~/src/app";
+import { getSignedCookieHeader } from "test/utils/cookies";
+
+describe("GitHub Create Branch Get", () => {
+	let app: Application;
+	beforeEach(() => {
+		app = express();
+		app.use((req, _, next) => {
+			req.log = getLogger("test");
+			req.csrfToken = jest.fn();
+			next();
+		});
+		app.use(getFrontendApp({
+			getSignedJsonWebToken: () => "",
+			getInstallationAccessToken: async () => ""
+		}));
+	});
+	describe("Testing the GET route", () => {
+		it("should redirect to Github login if unauthorized", async () => {
```

Comment:
Not sure you need to do this as it's already been tested in the github router

Reply:
This case was in the check list, so added a test case for this too. Leaving it as it is...